docs/man_pages/project/testing/debug-android.md

```diff
@@ -3,54 +3,42 @@ debug android
+Attaches the debug tools to a running app in the native emulator | `$ tns debug android --start --emulator [--timeout <timeout>]`
 
 Prepares, builds and deploys the project when necessary. Debugs your project on a connected device or emulator.
-While debugging, prints the output from the application in the console and watches for changes in your code. Once a change is detected, it synchronizes the change with all selected devices and restarts/refreshes the application.
+While debugging, prints the output from the application in the console and watches for changes in your code. Once a change is detected, it synchronizes the change with the selected device and restarts the application.
 
 ### Options
-* `--device` - Specifies a connected device on which to debug the app.
-* `--emulator` - Specifies that you want to debug the app in the native Android emulator from the Android SDK.
-* `--debug-brk` - Prepares, builds and deploys the application package on a device or in an emulator, launches the Chrome DevTools of your Chrome browser and stops at the first code statement.
+* `--device` - Specifies a connected device/emulator on which to debug the app.
+* `--emulator` - Specifies that you want to debug the app in the native Android emulator.
+* `--debug-brk` - Prepares, builds and deploys the application package on a device/emulator, generates a link for Chrome Developer Tools and stops at the first code statement.
 * `--start` - Attaches the debug tools to a deployed and running app.
-* `--stop` - Detaches the debug tools.
-* `--debug-port` - Sets a new port on which to attach the debug tools.
-* `--timeout` - Sets the number of seconds that the NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
+* `--timeout` - Sets the number of seconds that the NativeScript CLI will wait for the emulator/device to boot. If not set, the default timeout is 90 seconds.
 * `--no-watch` - If set, changes in your code will not be reflected during the execution of this command.
 * `--clean` - If set, forces rebuilding the native application.
 
 ### Attributes
-* `<Device ID>` is the index or name of the target device as listed by `$ tns device`
-* `<Port>` is an accessible port on the device to which you want to attach the debugging tools.
-* `<Emulator Options>` is any valid combination of options as listed by `$ tns help emulate android`
+* `<Device ID>` is the device identifier or name of the target device as listed by `$ tns device android`
 
 <% if(isHtml) { %>
-### Prerequisites
-
-* You must have Chrome installed on your system.<br/>If you are using a non-standard named Chrome app on an OS X system (for example, a nightly Canary update), you need to set this name in the `ANDROID_DEBUG_UI_MAC` setting in the NativeScript [config.json](file:///<%= #{config.getConfigPath(config)} %>).
```

Comment:
We should keep the `Prerequisites` with `You must have Chrome installed on your system`